DevCycle.SDK.Server.Local/Api/LocalBucketing.cs

```diff
@@ -525,7 +530,8 @@ private static string ReadAssemblyScriptString(Memory memory, int address)
         {
             // The byte length of the string is at offset -4 in AssemblyScript string layout.
             var length = memory.ReadInt32(address - 4);
-            return Encoding.Unicode.GetString(memory.GetSpan(address, length));
+            Span<byte> span = memory.GetSpan<byte>(address, length);
+            return Encoding.Unicode.GetString(span.ToArray());
```

Comment:
Instead of replacing these; I'd do a preprocessor check of 
```
#if NETSTANDARD_2_0
....
#endif
```

Reply:
fair, this is mostly me tinkering